src/pudl/metadata/fields.py

```diff
@@ -1970,6 +1982,15 @@
         "description": "The energy contained in fuel burned, measured in million BTU.",
         "unit": "MMBtu",
     },
+    "hour_of_year": {
+        "type": "number",
+        "description": "Number between 1 and 8670 representing the hour in a given year.",
+    },
+    "hour_utc": {
+        "type": "datetime",
+        "description": "Hourly datetime in UTC.",
+        "unit": "UTC",
```

Comment:
I don't think "UTC" makes sense as a unit, as the unit here is a measure of time not a timezone.